### PR TITLE
Feature/apicall

### PR DESCRIPTION
**Pull request by sydnerd**
# Pull Request Template
## Description

Fetch Bird data by state

## Type of change
- [ ] Bug fix 
- [X] New feature
- [ ] Refactoring

## Checklist:
- [X] My code follows the Turing's guidelines of this project
- [X] My changes generate no new errors/warnings

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -3,24 +3,36 @@ import NavBar from '../NavBar/NavBar';
...
 import StatePicker from '../StatePicker/StatePicker';
-
+import { getBirds } from '../../apiCalls';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-
+      selectedState: '',
```
Comment (LKessell) [inline src/components/App/App.js]: Now that we are passing the state abbreviation in to the setBirds call in StatePicker, we might not need to save this? Would love to hear your thoughts!

File: src/components/App/App.js [hunk 2]
```diff
@@ -3,24 +3,36 @@ import NavBar from '../NavBar/NavBar';
...
+      regionBirds: []
     }
   }
-  
+  // componentDidMount() {
+    
+  // }
+
+  selectState = (newState) => {
```
Comment (LKessell) [inline src/components/App/App.js]: See above comment, we may not need this anymore

File: src/components/StatePicker/StatePicker.js [hunk 3]
```diff
@@ -14,7 +15,8 @@ class StatePicker extends Component {
   
   handleSubmit = (event) => {
     event.preventDefault();
-    console.log('You selected' + this.state.value)
+    this.props.selectState(this.state.value)
```
Comment (LKessell) [inline src/components/StatePicker/StatePicker.js]: See above comments

File: src/components/App/App.js (hunk 1 above)
Comment (sydnerd) [reply src/components/App/App.js]: You're right, I think it's redundant

**Pull request merged**